app/models/tasks/cavc_remand_processed_letter_response_window_task.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+##
+# Task to indicate that Litigation Support is awaiting a response from the appellant,
+# after sending the CAVC-remand-processed letter to an appellant (SendCavcRemandProcessedLetterTask).
+# This task is for CAVC Remand appeal streams.
+# The appeal is put on hold for 90 days, with the option of ending the hold early.
+# After 90 days, the task comes off hold and show up in the CavcLitigationSupport team's unassigned tab
+# to be assigned and acted upon.
+# Expected parent: CavcTask
+# Expected assigned_to.type: CavcLitigationSupport
+
+class CavcRemandProcessedLetterResponseWindowTask < Task
+  validates :parent, presence: true, parentTask: { task_type: CavcTask }, on: :create
+
+  before_validation :set_assignee
+
+  def self.create_with_hold(parent_task)
+    multi_transaction do
+      create!(parent: parent_task, appeal: parent_task.appeal).tap do |window_task|
+        TimedHoldTask.create_from_parent(
+          window_task,
+          days_on_hold: 90,
+          instructions: ["This case is on hold for 90 days to allow the appellant to respond."]
+        )
+      end
+    end
+  end
+
+  def self.label
+    COPY::CAVC_REMAND_PROCESSED_LETTER_RESP_WINDOW_TASK_LABEL
+  end
+
+  def default_instructions
+    [COPY::CAVC_TASK_DEFAULT_INSTRUCTIONS]
+  end
+
+  USER_ACTIONS = [
+    Constants.TASK_ACTIONS.TOGGLE_TIMED_HOLD.to_h
+  ].freeze
+
+  def available_actions(_user)
+    USER_ACTIONS
```

Comment:
believe that if we do not check the user is a part of the organization here, we may be letting anyone in caseflow toggle the hold, will confirm in testing
```suggestion
  def available_actions(user)
    assigned_to.user_has_access?(user) ? USER_ACTIONS : []
```

Reply:
Yep! This user is not on the cavc team and has the toggle hold action
<img width="735" alt="Screen Shot 2020-11-23 at 3 49 17 PM" src="https://user-images.githubusercontent.com/45575454/100013851-82a98900-2da3-11eb-8002-f1b5dd028e1d.png">

Reply:
Also added test.